recipe/install_python.bat

```diff
@@ -0,0 +1,13 @@
+
+%PYTHON% setup.py build_ext --include-dirs %LIBRARY_INC% --library-dirs %LIBRARY_LIB% --gdal-config %LIBRARY_BIN%\gdal-config
+if errorlevel 1 exit 1
+
+%PYTHON% setup.py build_py
+if errorlevel 1 exit 1
+
+%PYTHON% setup.py build_scripts
+if errorlevel 1 exit 1
+
+%PYTHON% setup.py install
+if errorlevel 1 exit 1
+
```

Comment:
I guess we also need to remove `bld.bat`, right?

BTW, this is not how they did in AR, see https://github.com/AnacondaRecipes/gdal-feedstock/tree/master/recipe

Not sure if there is a reason or just style preference. Ping @msarahan here to comment on it.